FinSearchUnified/FinSearchUnified.php

```diff
@@ -34,14 +39,15 @@ public function build(ContainerBuilder $container)
             $container->setParameter($this->getContainerPrefix() . '.plugin_name', $this->getName());
         }
         if (!$container->has('shopware_search_es.product_number_search_factory')) {
-            $loader = new XmlFileLoader(
-                $container,
-                new FileLocator()
-            );
-
             $loader->load($this->getPath() . '/Resources/shopware/searchBundleES.xml');
         }
 
+        if ($container->getParameter('shopware.es.enabled')) {
+            $loader->load($this->getPath() . '/Resources/shopware/productSearch.xml');
+        } else {
+            $loader->load($this->getPath() . '/Resources/shopware/productSearch.xml');
+        }
```

Comment:
Shouldn't the first one be `productSearchES.xml`?
```suggestion
        if ($container->getParameter('shopware.es.enabled')) {
            $loader->load($this->getPath() . '/Resources/shopware/productSearchES.xml');
        } else {
            $loader->load($this->getPath() . '/Resources/shopware/productSearch.xml');
        }
```

Reply:
Oh, missed that while testing locally